### DISCUSSION TITLE
Version Change Documentation

### DISCUSSION THREAD

**Discussion by CodegoErgoSum** [community/community#69132] [New to GitHub]
Is there a tool, utility, facility, etc. that can document the changes between two different versions (specifically between one version to the next) in plain text, Word, PDF, spreadsheet or other human-readable format?

**Comment by queenofcorgis** [comment]
Hey there @CodegoErgoSum, 

I did a quick search for options for you: https://niftypm.com/blog/best-document-comparison-tools/#:~:text=Draftable%20is%20an%20amazing%20document,Draftable%20pro. I'd recommend going from there as GitHub (and our community) mostly focuses on version control which is the practice of tracking and managing changes to _software code_.
